x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -156,33 +163,35 @@ describe('Resolver, when analyzing a tree that has two related events for the or
     describe('when the related events button is clicked', () => {
       beforeEach(async () => {
         const button = await simulator.resolveWrapper(() =>
-          simulator.processNodeRelatedEventButton(entityIDs.origin)
+          simulator.processNodeChildElements(entityIDs.origin, 'resolver:submenu:button')
```

Comment:
why not just `testSubject` here? This couples the test to DOM structure. Is there a reason?

Reply:
I was just doing a one to one replacement that generalized this test a bit more. The only way to really not have this tied to the structure of the DOM would be to replace the `resolver:submenu:button` with a reference to the `entity-id`, maybe `resolver:node-${entityID}:submenu:button`. Is that what you're suggesting?

Reply:
Oh I see what you mean. I wasn't necessarily suggesting anything but here's my thoughts:
* The user doesn't care about the DOM structure and so if we don't rely on it our tests will be less brittle.
* It would be cool if we are only coupled to data-test attributes. That should make it simple to keep tests working. If we remove a wrapper div, I would hope that no tests break.

Maybe at our next office hours we can discuss those ideas more.

Reply:
I think there is some semantic weight to. e.g. the button being a _child_ / ("owned by" in the case of semantics) of the process node. I say keep it as is.

Reply:
Or to put it another way: If the relationship between the node and the interactive element it's expected to contain changes in some way, I think it's good that a test picks that up.